.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,6 @@
 TODO:
 
+- [ ] Documentation is updated or added in `packit/packit.dev`
```

Comment:
Thinking about it, we could also add „test coverage“

Reply:
Good point. Added 2 more items, and rephrased this one to make it sound more like a todo-item.

If this is good, I'll update the PR in packit/packit, too.